topics/climate/tutorials/climate-101/tutorial.md

```diff
@@ -0,0 +1,399 @@
+Freiburg). The data format may also have been changed (for instance to tabular) to ease processing. 
+
+## Get data
+
+> ### {% icon hands_on %} Hands-on: Data upload
+>
+> 1. Create a new history for this tutorial. If you are not inspired, you can name it *climate101*.
+>    {% include snippets/create_new_history.md %}
+> 2. Import the files from [Zenodo]() or from the shared data library
+>
+>    ```
+>    https://zenodo.org/record/3776500/files/tg_ens_mean_0.1deg_reg_v20.0e_Paris_daily.csv
+>    https://zenodo.org/record/3776500/files/ts_cities.csv
+>    ```
+>
+>    {% include snippets/import_via_link.md %}
+>    {% include snippets/import_from_data_library.md %}
+>
+> 3. Check that the datatype is **csv** or **netcdf** (files ending with `.nc`)
+>
+>    {% include snippets/change_datatype.md datatype="datatypes" %}
+>
+> 4. Add a tag to the dataset corresponding to `copernicus`
+>
+>    {% include snippets/add_tag.md %}
+>
+{: .hands_on}
+
+# What is climate?
+
+According to [wikipedia](https://en.wikipedia.org/wiki/Climate), 
+Climate is defined as the average state of everyday's weather condition **over a period of 30 years**. It is measured by assessing
+the patterns of variation in temperature, humidity, atmospheric pressure, wind, precipitation, atmospheric particle count and
+other meteorological variables in a given region over long periods of time.
+Climate differs from weather, in that weather only describes the short-term conditions of these variables in a given region. 
+
+
+## Climate versus Weather
+
+Quantities that climate scientists are interested in are similar to those used to assess the weather (temperature, precipitation, etc.). 
+But there is a big difference between climate and weather: **weather** varies from hour to hour and from day to day whereas **climate** 
+is defined as the average of weather over several decades or longer.
+
+The figure below shows a woman walking her dog and we can use it to make an analogy to illustrate the difference between weather and climate. 
+if you focus your attention on the dog, you can see that it is all over the place, sometimes upwards, sometimes downwards: this can represent the weather and its 
+variability. The dog (weather) is not following a fully random pattern and varies around a main direction (trend) that is given by the woman: the woman is representing
+the climate and gives us an indication of where both the woman and dog are likely to be in the future.
+  
+ ![Illustrate the difference beteen weather and climate](../../images/weather_versus_climate.png "Weather versus Climate")
+
+*Source: [Animated short on statistics](https://youtu.be/e0vj-0imOLw) from Norwegian infotainment program Siffer. Produced by TeddyTV for NRK. Animation by Ole Christoffer Haga*
+
+You can also watch this [Video](https://youtu.be/e0vj-0imOLw) to get an animated illustration of the difference between climate and weather.
```

Comment:
you can embed videos into the training with:

`<div class="embed-responsive embed-responsive-16by9"><iframe src="https://player.vimeo.com/video/120901536?portrait=0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe></div>` as seen here: https://raw.githubusercontent.com/galaxyproject/training-material/master/topics/introduction/tutorials/galaxy-intro-ngs-data-managment/tutorial.md